holmes/plugins/toolsets/kafka.py

```
@@ -352,17 +361,61 @@ def _invoke(self, params: Dict) -> StructuredToolResult:
+
+        # Check committed offsets for this consumer group on these topic partitions
+
+        committed_offsets = consumer.committed(topic_partitions, timeout=10.0)
+        consumer.close()
+
+        # Check if any partition has a valid committed offset
+        for tp in committed_offsets:
+            if tp.offset != -1001:  # -1001 means no committed offset
+                return True
+
+        return False
```

Ruff (0.11.9)

Use `return any(tp.offset != -1001 for tp in committed_offsets)` instead of `for` loop

Replace with `return any(tp.offset != -1001 for tp in committed_offsets)`

(SIM110)